### PR TITLE
Any Region Changer: ModMii Edition

### PR DESCRIPTION
**Pull request by GabubuAvailable**
- [x] Have you checked that there aren't other open pull requests for the same manifest update/change?
- [x] Have you verified that the manifest contains valid JSON?
- [x] Are you certain that this manifest is not designed to obtain copyrighted material or software, which you do not have permission to distribute?
- [x] Submitting malicious software is strictly forbidden, and potentially constitutes criminal activity, punishable by law. To the best of your knowledge, is this submission clear of any malicious intent?

Note: *filename*.oscmeta is the **slug** of the application you are submitting. This slug is used to locate files such as "/apps/**slug**/boot.dol" and "/apps/**slug**/meta.xml" in the final archive generated by this manifest. These files should be available at this location.

---

Any Region Changer: ModMii Edition, a tool used for region changing the Wii console with ModMii software. What else should I say? lol

### CODE REVIEW

File: contents/ARCME.oscmeta [hunk 1]
```diff
@@ -0,0 +1,15 @@
+{
+    "information": {
+        "name": "Any Region Changer: ModMii Edition",
+        "author": "Various, bushing, tona, scooby74029, TheShadowEevee, XFlak",
```
Comment (emilydaemon) [inline contents/ARCME.oscmeta:4]: Is the author field in the meta.xml this long? I think it could be trimmed down to just "Various".

**Comment by dhtdht020** [comment]
I've seen concerns on the potential for bricks from inexperienced users. Are we sure this should be added?

**Comment by GabubuAvailable** [comment]
The app will show a warning message once they will launch it. _but idk if some of them will read that_

**Comment by dhtdht020** [comment]
Warning seems good enough, not our role to babysit users.

**Comment by GabubuAvailable** [comment]
Agreed

**Comment by dhtdht020** [comment]
we might want to implement a warning flag first, do not merge for now.

**Comment by GabubuAvailable** [comment]
I think you should also do that with other apps which modifies the NAND.

**Comment by dhtdht020** [comment]
alright, we can look into implementing a "writes to nand" flag in the future, but for now I do not see the issue with merging.

If this is still worrysome to anyone, potentially a short description override that says "Use with caution!" could be applied.

**Pull request merged**